lib/cryptoexchange/exchanges/deex/services/order_book.rb

```diff
@@ -0,0 +1,46 @@
+module Cryptoexchange::Exchanges
+  module Binance
```

Comment:
how come binance is here.

Reply:
@estherleongym probably want to fix this?

Reply:
@estherleongym can i somehow contact you? Will you edit it?